### PR TITLE
Add "--assessment" flag, order Redshift raw logs if it is enabled

### PR DESCRIPTION
**Pull request by frsv**
This clashes a bit with #77 in terms of flag definition. I will reuse the code from it if it is merged earlier

### CODE REVIEW

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java [hunk 1]
```diff
@@ -132,7 +132,7 @@ public class ConnectorArguments extends DefaultArguments {
     private final OptionSpec<String> optionWarehouse = parser.accepts(OPT_WAREHOUSE, "Virtual warehouse to use once connected (for providers such as Snowflake)").withRequiredArg().ofType(String.class);
     private final OptionSpec<String> optionDatabase = parser.accepts(OPT_DATABASE, "Database(s) to export").withRequiredArg().ofType(String.class).withValuesSeparatedBy(',').describedAs("db0,db1,...");
     private final OptionSpec<String> optionSchema = parser.accepts(OPT_SCHEMA, "Schemata to export").withRequiredArg().ofType(String.class).withValuesSeparatedBy(',').describedAs("sch0,sch1,...");
-    private final OptionSpec<Void> optionAssessment = parser.accepts(OPT_ASSESSMENT, "Whether to create a dump for assessment (i.e., dump additional information).");
+    private final OptionSpec<Boolean> optionAssessment = parser.accepts(OPT_ASSESSMENT, "Whether to create a dump for assessment (i.e., dump additional information).").withOptionalArg().withValuesConvertedBy(BooleanValueConverter.INSTANCE).defaultsTo(Boolean.FALSE);;
```
Comment (shevek-google) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java:135]: double semicolon

**Review by shevek-google — APPROVED** [review]
I think this could be a tiny bit cleaner, but I approve.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java (hunk 1 above)
Comment (frsv) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java]: Done.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftRawLogsConnector.java [hunk 2]
```diff
@@ -83,17 +85,31 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
 
         // DDL TEXT is simple ...
         // min() as there is no ANY() or SOME()
-        String queryTemplateDDL = "SELECT userid, xid, pid, trim(label) as label, starttime, endtime, sequence, text FROM STL_DDLTEXT WHERE ##";
-        makeTasks(arguments, intervals, RedshiftRawLogsDumpFormat.DdlHistory.ZIP_ENTRY_PREFIX, queryTemplateDDL, "starttime", parallelTask);
+        StringBuilder queryTemplateDDL = new StringBuilder(
+            "SELECT userid, xid, pid, trim(label) as label, starttime, endtime, sequence, text FROM STL_DDLTEXT WHERE ##");
+
+        if (arguments.isAssessment()) {
+            queryTemplateDDL.append(" ORDER BY starttime, xid, pid, sequence");
```
[the remark this answers is not in the document]
Comment (frsv) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftRawLogsConnector.java]: Done.

**Pull request merged**